cypress/features/regression/alertActions.feature

```diff
@@ -1,25 +1,9 @@
 Feature: Alert component actions
-  I want to check Alert component actions
-
-  Background: Open Alert component page
-    Given I open "Alert Test" component page "default"
-
-  @positive
-  Scenario: Clicking close icon, closes Alert dialog
-    # commented because of BDD default scenario Given - When - Then
-    # When I open "Alert Test" component page "default"
-    Then closeIcon is visible in iframe
-      And I click closeIcon in IFrame
-      And Alert is not visible
-
-  @positive
-  Scenario: Disable escape key
```

Comment:
why we deleted this TC ?

Reply:
it's the same TC as in `alert.feature` file